### ISSUE TITLE
vagrant_2.1.2_i686.msi does not install vagrant.exe

### ISSUE THREAD

**Issue by Jaimy85** [hashicorp/vagrant#10026]
### Vagrant version
2.1.2

### Host operating system
Windows 10 32 bit

### Expected behavior
Install vagrant.exe in C:\HashiCorp\Vagrant\bin

### Actual behavior
folder is empty, vagrant.exe is absent

### Steps to reproduce
1. clean install vagrant 2.1.2
2. upgrade from 2.1.1

Clean install of previous version 2.1.1 works without a problem.

**Comment by emailwedgy** [comment]
+1 
New to Vagrant, this took me ages to identify. Thanks Jaimy85

**Comment by brupelo** [comment]
After spending a fair ammount of time trying the latest 2.1.2 I've concluded the installer is broken, good this issues confirms it. In my case, the whole thing broke after trying to upgrade from 1.9.x.

It'd be cool if you guys improved the windows installer. Not only is a painfully slow experience uninstalling/installing vagrant but for some obscure reason each time I've tried upgrading vagrant in the past few years each times something broke consistently. Even if following the best practices (uninstall+restart) and then (install+restart).

Thanks.
